staging/kos/pkg/apis/network/v1alpha1/types.go

```diff
@@ -45,12 +43,12 @@ type ObjectWrite struct {
 	Section string `json:"section" protobuf:"bytes,1,name=section"`
 
 	// ServerTime is the time when the write was recorded at the apiserver
-	ServerTime Timestamp `json:"serverTime" protobuf:"bytes,2,name=serverTime"`
+	ServerTime metav1.MicroTime `json:"serverTime" protobuf:"bytes,2,name=serverTime"`
 }
 
 // GetServerWriteTime returns the server time of the write to the
 // given section, or zero if there was none.
-func (writes WriteSet) GetServerWriteTime(section string) Timestamp {
+func (writes WriteSet) GetServerWriteTime(section string) metav1.MicroTime {
```

Comment:
At the cost of being inconsistent with the rest of this file, this could return a plain `time.Time`.
This way callers of the function can avoid appending `.Time` to the return argument.

Reply:
I added a function rather than modify this one.